### PR TITLE
Add support for qualified idents

### PR DESCRIPTION
**Pull request by just-sultanov**
Added support (search, add, update, delete) for qualified idents (keys, values)

Signed-off-by: Ilshat Sultanov <just.sultanov@gmail.com>

### CODE REVIEW

File: src/clucie/utils.clj [hunk 1]
```diff
@@ -1,8 +1,26 @@
 (ns clucie.utils)
 
+(defn kw-or-symbol? [x]
```
Comment (federkasten) [inline src/clucie/utils.clj]: May be better to name `keyword-or-symbol?`.

**Review by federkasten — CHANGES_REQUESTED** [review]
Basically looks good 👍 
I just leave a comment for function naming.

File: src/clucie/utils.clj (hunk 1 above)
Comment (just-sultanov) [reply src/clucie/utils.clj]: @federkasten done 👍

**Review by federkasten — APPROVED** [review]

**Comment by federkasten** [comment]
Thanks @just-sultanov  !

**Comment by federkasten** [comment]
Released in [0.4.2](https://clojars.org/clucie/versions/0.4.2)

**Pull request merged**